forge-gui/res/cardsfolder/upcoming/aberrant.txt

```diff
@@ -0,0 +1,11 @@
+Name:Aberrant
+ManaCost:X 1 G
+Types:Creature Tyranid Mutant
+PT:0/0
+K:Ravenous
+K:Trample
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | Execute$ TrigDestroy | CombatDamage$ True | TriggerDescription$ Heavy Power Hammer — Whenever CARDNAME deals combat damage to a player, destroy target artifact or enchantment that player controls.
+SVar:TrigDestroy:DB$ Destroy | ValidTgts$ Artifact.ControlledBy TriggeredDefendingPlayer,Enchantment.ControlledBy TriggeredDefendingPlayer | TgtPrompt$ Select target artifact or enchantment that player controls
```

Comment:
such cards should probably use `TriggeredDamageTarget` because the damage might get redirected to another player